platform/cosmos/stake.go

```diff
@@ -38,15 +42,33 @@ func (p *Platform) GetValidators() (blockatlas.ValidatorPage, error) {
 }
 
 func (p *Platform) GetDetails() blockatlas.StakingDetails {
-	//TODO: Find a way to have a dynamic
 	return blockatlas.StakingDetails{
-		Reward:        blockatlas.StakingReward{Annual: 11},
+		Reward: blockatlas.StakingReward{
+			Annual: p.GetMaxAPR(),
+		},
 		MinimumAmount: blockatlas.Amount("0"),
 		LockTime:      1814400,
 		Type:          blockatlas.DelegationTypeDelegate,
 	}
 }
 
+func (p *Platform) GetMaxAPR() float64 {
+	validators, err := p.GetValidators()
+	if err != nil {
+		return defaultAnnualReward
```

Comment:
`defaultAnnualReward` make it global for all staking platforms

Reply:
Do we need to use it (defaultAnnualReward) with Algorand, TRON, Tezos?